WooCommerce/Classes/ViewRelated/Products/Edit Product/Product Settings/ProductSettingsSections.swift

```diff
@@ -20,12 +20,13 @@ enum ProductSettingsSections {
 
         let rows: [ProductSettingsRowMediator]
 
-        init(_ settings: ProductSettings, productType: ProductType, isEditProductsRelease3Enabled: Bool) {
-            if isEditProductsRelease3Enabled && productType == .simple {
+        init(_ settings: ProductSettings, productType: ProductType, isEditProductsRelease3Enabled: Bool, isEditProductsRelease5Enabled: Bool) {
+            if (isEditProductsRelease3Enabled || isEditProductsRelease5Enabled) && productType == .simple {
```

Comment:
I'd say that this condition is wrong. We don't want to show the downloadable product row if only the product release 3 flag is enabled.

Reply:
Hi @pmusolino , I think I didn't quite get your point. Can you please elaborate more on when do we want to show the downloadable product row and when we don't

Reply:
Updated the logic. Is it OK now?